apps/api/v2/src/modules/organizations/services/organizations-users-service.ts

```diff
@@ -79,8 +79,6 @@ export class OrganizationsUsersService {
       updateUserBody
     );
 
-    await this.organizationsTeamsService.addUserToTeamEvents(user.id, org.id);
```

Comment:
we don't want to add users to team event-types when they are invited to the Org, only when they are invited to a team